db/seeds/base/add_applications.rb

```diff
@@ -75,13 +75,80 @@
       :application,
       4,
       :withdrawn,
+      %i[accepted rejected].sample,
       :with_random_user,
       :with_random_work_setting,
-      :with_random_lead_provider_approval_status,
       :with_random_participant_outcome_state,
       lead_provider:,
       course: Course.all.sample,
       cohort: Cohort.all.sample,
     )
+
+    # users with one eligible for funded place application each (cohort funding_cap true)
```

Comment:
it feels like we're adding a bit too many applications here, and we're not looking at the withdrawn statuses yet in this ticket. Could we just maybe add 2 applications in a cohort with funded cap set with true/false and accepted so they are valid and maybe that's enough?